libs/usb-drive/src/usb_drive.ts

```diff
@@ -152,7 +153,100 @@ function isFat32(deviceInfo: BlockDeviceInfo): boolean {
   return deviceInfo.fstype === 'vfat' && deviceInfo.fsver === 'FAT32';
 }
 
-export function detectUsbDrive(): UsbDrive {
+async function logMountInit(logger: Logger): Promise<void> {
```

Comment:
Separating out the logging as methods to improve readability of the main functions.

Reply:
We could take this one step further by decorating functions with logging, since the pattern is always the same.

A sketch of how this would look:
```ts
function withLogging(actionLabel, func) {
  try {
    logger.log(actionLabel, 'init');
    await func();
    logger.log(actionLabel, 'success');
  } catch (error) {
    logger.log(actionLabel, 'failure', error);
    throw error;
  }
}

const mount = withLogging(UsbDriveMount, () => {... mount implementation ...});
```